lib/verdict.rb

```diff
@@ -20,7 +20,11 @@ def repository
   end
 
   def discovery
-    Dir[File.join(Verdict.directory, '**', '*.rb')].each { |f| require f } if @directory
+    Dir[File.join(Verdict.directory, '**', '*.rb')].each { |f| load f } if @directory
```

Comment:
I like the idea of using (multiple) constants to load experiment definitions, but it'd require major changes. Although it'd be cleaner and more railsy, I don't think it's worth the efforts.

@rafaelfranca explained that this (PR) is the right thing to do, as Verdict's experiment definitions are akin to rake tasks.